src/vehicle_move.cpp

```diff
@@ -804,6 +804,10 @@ void vehicle::handle_trap( const tripoint &p, int part )
     const trap &tr = g->m.tr_at( p );
     const trap_id t = tr.loadid;
 
+    if( t == tr_null ) {
```

Comment:
Technically speaking, there IS a trap with `tr_null` id. It doesn't do anything, but it exists. To be sure in non-existence of any traps you can use `tr.is_null()` check.

Reply:
`tr.is_null()` is `return loadid == tr_null;` so there is no difference in functionality between `t == tr_null` and calling `tr.is_null()`

Reply:
Ok, got it.